mlflow/cli.py

```diff
@@ -250,8 +274,7 @@ def server(backend_store_uri, default_artifact_root, host, port,
             sys.exit(1)
 
     try:
-        _run_server(backend_store_uri, default_artifact_root, host, port, workers, static_prefix,
```

Comment:
I think just adding waitress_opts is easier to read than passing in **server_kwargs.